tests/ResponseGrabberTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Testing\Tests;
+
+use Nyholm\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+use RuntimeException;
+use Yiisoft\Yii\Testing\ResponseAccessor;
+use Yiisoft\Yii\Testing\ResponseGrabber;
+
+class ResponseGrabberTest extends TestCase
+{
+    public function testGetResponseWillReturnResponseAccessorSetResponse(): void
+    {
+        $response = new Response(200, body: '{"key": "value"}', reason: 'Ok with body', version: '1.1');
+
+        $responseGrabber = new ResponseGrabber();
+        $responseGrabber->setResponse($response);
+
+        $this->assertEquals(new ResponseAccessor($response), $responseGrabber->getResponse());
+    }
+
+    public function testGetResponseWillThrowExceptionIfSetResponseIsCalledWithNullParameter(): void
+    {
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('Response is null');
+
+        $responseGrabber = new ResponseGrabber();
+        $responseGrabber->setResponse(null);
+        $responseGrabber->getResponse();
+    }
+
+    public function testGetResponseWillThrowExceptionIfResponseIsNotSet(): void
+    {
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('Response is null');
+
+        (new ResponseGrabber())->getResponse();
+    }
```

Comment:
These tests do not correspond to real usage. Would you please make it closer to real usage?